server/views/pages/incentives-table.njk

```diff
@@ -69,13 +69,13 @@
             {% set _ = rows.push([
             { html: '<img src="' + imageUrl + '" alt="' + prisoner.lastName + ', ' + prisoner.firstName + '\'s photo" width="90" />' },
             { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '" class="govuk-link">' + prisoner.lastName + ', ' + prisoner.firstName + ' (' + prisoner.prisonerNumber + ')</a>' },
-            { text: prisoner.daysOnLevel },
-            { text: prisoner.daysSinceLastReview },
-            { text: prisoner.positiveBehaviours },
-            { text: prisoner.incentiveEncouragements },
-            { text: prisoner.negativeBehaviours },
-            { text: prisoner.incentiveWarnings },
-            { text: prisoner.provenAdjudications }
+            { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '/incentive-level-details?agencyId=' + entries.prisonId + '&incentiveLevel=' + level.levelDescription + '" class="govuk-link">' + prisoner.daysOnLevel + '</a>' },
+            { html: '<a href="' + dpsHome + '/prisoner/' + prisoner.prisonerNumber + '/incentive-level-details?agencyId=' + entries.prisonId + '&incentiveLevel=' + level.levelDescription + '" class="govuk-link">' + prisoner.daysSinceLastReview + '</a>' },
```

Comment:
This link and the one above are the same (AFAICT) - maybe it's worth having it in a variable - or maybe it's overkill.